app/views/revisions/index.slim

```diff
@@ -81,7 +92,30 @@ div.content-container
               td.empty-cell
                 span.sr-only Cell empty because only a metadata revision
             -else
-              td ==link_to_revision(rev)
+              td.revision-label-cell
+                div.mouseoveredit
+                  ==render partial: 'link', locals: { revision: rev }
+                  span.sr-only #{" "}
+                  span.hidden aria-hidden="true" #{"["}
+                  a.edit-icon.tooltips href="#" title="Edit revision label"
+                    span.glyphicon.glyphicon-pencil aria-hidden="true"
+                    span.sr-only Edit revision label
+                  span.hidden aria-hidden="true" #{"]"}
+                form.revision-label-edit-form style="display: none" method="patch" data-url="#{revision_path(rev)}"
+                  -if rev.friendly_label != nil
```

Comment:
Why do you need this differentiation. if rev.friendly_label is nil, it will just populate with empty values, no?  Just want to make sure we don't have an unnecessary if/then block that makes two almost identical lines sets of code for no reason.

Reply:
My bad. Yep. You're right. For some reason I thought a `nil` value in a string would cause an error.